<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_DlgOptionsGeneral</class>
 <widget class="QDialog" name="QG_DlgOptionsGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Application Preferences</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="Widget9">
      <attribute name="title">
       <string>&amp;Appearance</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="Q3ButtonGroup" name="bgLanguage">
         <property name="title">
          <string>Language</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lLanguage">
            <property name="text">
             <string>&amp;GUI Language:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>cbLanguage</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lLanguageCmd">
            <property name="text">
             <string>&amp;Command Language:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>cbLanguageCmd</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cbLanguageCmd"/>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbLanguage"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="Q3ButtonGroup" name="bgColors">
         <property name="title">
          <string>Colors</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lBackground">
            <property name="text">
             <string>Backgr&amp;ound:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>cbBackgroundColor</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lGridColor">
            <property name="text">
             <string>G&amp;rid Color:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>cbGridColor</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbBackgroundColor">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>#000000</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>#ffffff</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>#c0c0c0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>#808080</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cbGridColor">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>#c0c0c0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>#ffffff</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>#000000</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>#808080</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lMetaGridColor">
            <property name="text">
             <string>&amp;Meta Grid Color:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>cbMetaGridColor</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cbMetaGridColor">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>#404040</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>#000000</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>#ffffff</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>#c0c0c0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>#808080</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lSelectedColor">
            <property name="text">
             <string>S&amp;elected Color:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>cbMetaGridColor</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cbSelectedColor">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>#a54747</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="cbHighlightedColor">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>#739373</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lHighlightedColor">
            <property name="text">
             <string>&amp;Highlighted Color:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>cbMetaGridColor</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <spacer name="spacer7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>21</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="1">
           <spacer name="spacer8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>21</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="Q3ButtonGroup" name="buttonGroup5_2">
         <property name="title">
          <string>Fontsize</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lSizeStatus">
            <property name="text">
             <string>Statusbar:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbSizeStatus">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>11</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="spacer11_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <spacer name="spacer12">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="Q3ButtonGroup" name="bgGraphicView">
         <property name="title">
          <string>Graphic View</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="cbShowCrosshairs">
            <property name="text">
             <string>&amp;Show large crosshairs</string>
            </property>
            <property name="shortcut">
             <string>Alt+S</string>
            </property>
            <attribute name="buttonGroup">
             <string/>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="cbScaleGrid">
            <property name="text">
             <string>A&amp;utomatically scale grid</string>
            </property>
            <property name="shortcut">
             <string>Alt+U</string>
            </property>
            <attribute name="buttonGroup">
             <string/>
            </attribute>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="spacer29">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>21</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="1">
           <spacer name="spacer29_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>21</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lMaxPreview">
            <property name="text">
             <string>Number of p&amp;review entities:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>cbMaxPreview</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cbMaxPreview">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>200</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lMinGridSpacing">
            <property name="text">
             <string>Minimal Grid Spacing (px):</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>cbMaxPreview</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cbMinGridSpacing">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="spacer9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>31</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <spacer name="spacer4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>31</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Paths</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="textLabel2">
         <property name="text">
          <string>Translations:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="textLabel3">
         <property name="text">
          <string>Hatch Patterns:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="textLabel4">
         <property name="text">
          <string>Fonts:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="textLabel5">
         <property name="text">
          <string>Scripts:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="textLabel6">
         <property name="text">
          <string>Part Libraries:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lePathTranslations"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lePathHatch"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lePathFonts"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lePathScripts"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lePathLibrary"/>
       </item>
       <item row="5" column="0">
        <spacer name="spacer5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>31</width>
           <height>71</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="1">
        <spacer name="spacer5_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>31</width>
           <height>71</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Defaults</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="Q3ButtonGroup" name="buttonGroup5">
         <property name="title">
          <string>Defaults for new drawings</string>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="lUnit">
            <property name="text">
             <string>&amp;Unit:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>cbUnit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbUnit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="spacer11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>21</width>
           <height>61</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="spacer11_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>211</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="Horizontal Spacing2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string>Alt+O</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>cbLanguage</tabstop>
  <tabstop>cbLanguageCmd</tabstop>
  <tabstop>cbShowCrosshairs</tabstop>
  <tabstop>cbScaleGrid</tabstop>
  <tabstop>cbMinGridSpacing</tabstop>
  <tabstop>cbMaxPreview</tabstop>
  <tabstop>cbBackgroundColor</tabstop>
  <tabstop>cbGridColor</tabstop>
  <tabstop>cbMetaGridColor</tabstop>
  <tabstop>cbSelectedColor</tabstop>
  <tabstop>cbHighlightedColor</tabstop>
  <tabstop>cbSizeStatus</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>lePathTranslations</tabstop>
  <tabstop>lePathHatch</tabstop>
  <tabstop>lePathFonts</tabstop>
  <tabstop>lePathScripts</tabstop>
  <tabstop>lePathLibrary</tabstop>
  <tabstop>cbUnit</tabstop>
 </tabstops>
 <includes>
  <include location="local">rs_settings.h</include>
  <include location="local">rs_system.h</include>
  <include location="local">rs.h</include>
  <include location="local">rs_units.h</include>
 </includes>
 <resources>
  <include location="png/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>QG_DlgOptionsGeneral</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>QG_DlgOptionsGeneral</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSizeStatus</sender>
   <signal>activated(int)</signal>
   <receiver>QG_DlgOptionsGeneral</receiver>
   <slot>setRestartNeeded()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbLanguageCmd</sender>
   <signal>activated(int)</signal>
   <receiver>QG_DlgOptionsGeneral</receiver>
   <slot>setRestartNeeded()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbLanguage</sender>
   <signal>activated(int)</signal>
   <receiver>QG_DlgOptionsGeneral</receiver>
   <slot>setRestartNeeded()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
